api/RTCPeerConnection.json

```diff
@@ -1394,6 +1395,7 @@
       "getLocalStreams": {
         "__compat": {
           "mdn_url": "https://developer.mozilla.org/docs/Web/API/RTCPeerConnection/getLocalStreams",
+          "spec_url": "https://w3c.github.io/webrtc-pc/#dom-peerconnection-getsenders",
```

Comment:
This looks wrong

Reply:
`getLocalStreams` is deprecated and standard_track:false, and not a member of RTCPeerConnection in the spec, and https://developer.mozilla.org/docs/Web/API/RTCPeerConnection/getLocalStreams redirects to https://developer.mozilla.org/en-US/docs/Web/API/RTCPeerConnection/getSenders — so I’ve updated the patch to drop the `spec_url` here.